src/main/java/seedu/duke/command/CommandResult.java

```diff
@@ -6,8 +6,4 @@
     public CommandResult(String feedbackToUser) {
         this.feedbackToUser = feedbackToUser;
     }
-
-    public String getFeedbackToUser() {
```

Comment:
If removing getter function, then the `feedbackToUser` attribute should not be private.